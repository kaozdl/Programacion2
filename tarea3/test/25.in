# buscar_camino
insertar_binario (15,q) .
buscar_camino (15,q) .
buscar_camino (18,q) .
insertar_binario (12,d) .
insertar_binario (18,o) (21,v) .
insertar_binario (30,t) .
buscar_camino (15,q) (21,v) (30,t) .
buscar_camino (21,q) (55,v) (18,t) .
buscar_camino (15,a) (21,v) (30,t) .
buscar_camino (15,q) (21,v) (30,c) .
buscar_camino (12,v) (18,t) .
buscar_camino (12,q) (18,d) (-2,o) .
buscar_camino (12,q) (18,d) .
buscar_camino (12,q) (18,d) (-2,o) (5,y) .
Fin
