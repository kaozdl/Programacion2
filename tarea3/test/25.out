1>
2>
(15,q)
0
0
3>0-()
4>0-()
5>
(15,q)
-(12,d)
1
0
6>
-(21,v)
(15,q)
--(18,o)
-(12,d)
2
1
7>
-(21,v)
--(30,t)
(15,q)
--(18,o)
-(12,d)
2
2
8>2-(D,I)
9>2-(D,I)
10>No existe
11>No existe
12>No existe
13>2-(I,D)
14>No existe
15>No existe
16>Fin.
